.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,75 @@
+name: Build wheels
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '0 0 * * 0,3' # 2/weekly
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        os: [ubuntu-latest, macos-latest]
+        platform: [x64]
+    env:
+      PKG_NAME: gensim
+      REPO_DIR: gensim
+      BUILD_COMMIT: HEAD
+      PLAT: x86_64
+      UNICODE_WIDTH: 32
+      MB_PYTHON_VERSION: ${{ matrix.python-version }} # MB_PYTHON_VERSION is needed by Multibuild
+      TEST_DEPENDS: Morfessor==2.0.2a4 python-levenshtein==0.12.0 visdom==0.1.8.9 pytest mock cython nmslib pyemd testfixtures scikit-learn pyemd
+      DOCKER_TEST_IMAGE: multibuild/xenial_x86_64
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+        fetch-depth: 0
+    - name: Setup OSX var for Multibuild
+      if: ${{ matrix.os == 'macos-latest' }}
+      run: |
+        echo "TRAVIS_OS_NAME=osx" >> $GITHUB_ENV;
+    - name: Setup SKIP_NETWORK_TEST env variable
+      if: ${{ matrix.python-version != 3.6 || matrix.os != 'ubuntu-latest' }}
+      run: |
+        echo "SKIP_NETWORK_TESTS=1" >> $GITHUB_ENV;
+    - name: Print some Environment variable
+      run: |
+        echo "PLAT: ${PLAT}"
+        echo "DOCKER_TEST_IMAGE: ${DOCKER_TEST_IMAGE}"
+        echo "TEST_DEPENDS: ${TEST_DEPENDS}"
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install VirtualEnv
+      run: |
+        python -m pip install --upgrade pip
+        pip install virtualenv
+    - name: Build and Install Wheels
+      run: |
+        source multibuild/common_utils.sh
+        source multibuild/travis_steps.sh
+        source config.sh
+        echo "------ BEFORE INSTALL ---------"
```

Comment:
Instead of doing this echo-debugging ourselves, is it possible to make these things different steps in the workflow?

e.g.

```
- name: multibuild before_install
  run: |
        source multibuild/common_utils.sh
        source multibuild/travis_steps.sh
        source config.sh
        before_install

- name: multibuild build_wheel
  run: |
        source multibuild/common_utils.sh
        source multibuild/travis_steps.sh
        source config.sh
        build_wheel

- name: multibuild install_run
  run: |
        source multibuild/common_utils.sh
        source multibuild/travis_steps.sh
        source config.sh
        install_run
```